todo-app/src/components/Todos/List-todos.js

```diff
@@ -0,0 +1,69 @@
+// jshint esnext: true
+
+import React from 'react';
+import TodoItem from './Todo-item';
+
+
+let todosInfo = [
+    {
+        id: 1,
+        author: 'person 1',
+        date: new Date().toLocaleString(),
```

Comment:
It would be better to just set our date as `new Date` here, and use `toLocaleString` when we want to display it in some UI. Maintaining the full date object as opposed to immediately converting it to a string allows us much more flexibility.